data/brands/amenity/post_office.json

```diff
@@ -139,6 +139,23 @@
         "short_name": "UPS"
       }
     },
+    {
+      "displayName": "中国邮政",
+      "id": "chinapost-1c81b5",
+      "locationSet": {"include": ["cn"]},
+      "matchNames": ["ems", "邮政", "邮政速递", "中国邮政速递", "中国邮政速递物流"],
+      "tags": {
+        "amenity": "post_office",
+        "name": "中国邮政",
+        "name:en": "China Post",
+        "name:zh": "中国邮政",
+        "brand": "中国邮政",
+        "brand:en": "China Post",
+        "brand:wikidata": "Q1066476",
+        "brand:wikipedia": "zh:中国邮政",
+        "brand:zh": "中国邮政"
```

Comment:
```suggestion
        "brand:zh": "中国邮政",
        "operator":"中国邮政集团公司",
        "operator:zh":"中国邮政集团公司",
        "operator:en":"China Post Corporation",
        "operator:wikidata":"Q1066476",
        "operator:wikipedia":"zh:中国邮政"
```

Reply:
我的意见倒是非常模糊：之前的民营快递，挪了是很正常的，因为不同地区有不同的承包商。菜鸟驿站也是一样。
但是中国邮政，可以确定是由中国邮政集团公司China Post Corporation运营的一项公共基础服务。（此外还可与中华邮政保持一致）。但具体到运营商，到底什么层次才算不同operator，我也拿捏不准。
因此两种做法我皆认为有道理。因此我在本PR里不做最终决断。不过如果决定合并，则希望能追加上述内容。

Machine Translation:

My opinion is very vague: the previous private express delivery is normal, because there are different contractors in different regions. The same is true for the rookie station.
However, China Post can be determined to be a public basic service operated by China Post Corporation. (In addition, it can be consistent with Chunghwa Post). But when it comes to operators, I am not sure what level is considered to be a different operator.
Therefore, I think both approaches make sense. Therefore, I will not make the final decision in this PR. However, if you decide to merge, I hope to add the above content.